src/app/layout.tsx

```diff
@@ -18,8 +18,8 @@ export default function RootLayout({
       <body>
         <ThemeProvider>
           <Header />
-          {children}
         </ThemeProvider>
+        <main>{children}</main>
```

Comment:
I believe this should be part of `ThemeProvider`?

Reply:
the themeProvider was only used in header part. I thought its okay to not include main